icu4c/source/i18n/unicode/timezone.h

```diff
@@ -223,21 +233,54 @@ class U_I18N_API TimeZone : public UObject {
      * @param rawOffset an offset from GMT in milliseconds, ignoring
      * the effect of daylight savings time, if any
      * @return an enumeration object, owned by the caller
-     * @stable ICU 2.4
+     * @deprecated ICU 70 Use createEnumerationForOffset(int32_t,UErrorCode&) instead.
      */
     static StringEnumeration* U_EXPORT2 createEnumeration(int32_t rawOffset);
 
+    /**
+     * Returns an enumeration over time zone IDs with a given raw
+     * offset from GMT.  There may be several times zones with the
+     * same GMT offset that differ in the way they handle daylight
+     * savings time.  For example, the state of Arizona doesn't
+     * observe daylight savings time.  If you ask for the time zone
+     * IDs corresponding to GMT-7:00, you'll get back an enumeration
+     * over two time zone IDs: "America/Denver," which corresponds to
+     * Mountain Standard Time in the winter and Mountain Daylight Time
+     * in the summer, and "America/Phoenix", which corresponds to
+     * Mountain Standard Time year-round, even in the summer.
+     *
+     * @param rawOffset an offset from GMT in milliseconds, ignoring
+     * the effect of daylight savings time, if any
+     * @param status Receives the status.
+     * @return an enumeration object, owned by the caller
+     * @stable ICU 70
+     */
+    static StringEnumeration* U_EXPORT2 createEnumerationForOffset(int32_t rawOffset, UErrorCode& status);
```

Comment:
```suggestion
    static StringEnumeration* U_EXPORT2 createEnumerationForRawOffset(int32_t rawOffset, UErrorCode& status);
```
... For *Raw* Offset as discussed

Reply:
Done. I misunderstood the conclusion - sorry.